lib/resource-server.js

```diff
@@ -59,8 +61,7 @@ function setupResourceServer(app, options, models) {
             }
             if (!u) {
               return done(
-                new TokenError('Access token has invalid user id: ' +
-                  userId, 'invalid_grant'));
+                new TokenError(g.f('Access token has invalid user id: %s invalid_grant', userId)));
```

Comment:
This doesn't need to be within curly brackets but the whole thing needs correction to 
```
new TokenError(g.f('Access token has invalid user id: %s', userId), 'invalid_grant'));
```